recipes/cupy-xarray/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "cupy-xarray" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cupy-xarray-{{ version }}.tar.gz
+  sha256: fcaf71201fa75bfbb6008740787ee0c34b1ad83d7ca381e2dd915a5e6af7b3c6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```

Skip  builds for unsupported platforms? This is taken from the cupy feedstock

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  skip: true  # [(not win64 and not linux and not (aarch64 and arm_variant_type == "sbsa")) or cuda_compiler_version in (undefined, "None")]
```

---

Think we can go with `noarch` here and leave out the `skip`. Since dependencies already will or won't be available on different platforms, Conda will already raise an error about missing dependencies.